packages/herdtools7/herdtools7.7.56.1/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+name: "herdtools7"
+version: "7.56.1"
+synopsis: "The herdtools suite for simulating and studying weak memory models"
+description: ""
+maintainer: "Luc Maranget <Luc.Maranget@inria.fr>"
+authors: [
+  "Luc Maranget <Luc.Maranget@inria.fr>"
+  "Jade Alglave <j.alglave@ucl.ac.uk>"
+]
+homepage: "http://diy.inria.fr/"
+bug-reports: "http://github.com/herd/herdtools7/issues/"
+doc: "http://diy.inria.fr/doc/index.html"
+dev-repo: "git+https://github.com/herd/herdtools7.git"
+build: ["sh" "./dune-build.sh" "%{prefix}%"]
+install: ["sh" "./dune-install.sh" "%{prefix}%"]
+# @todo Add "build-doc" field
+# @todo Add "build-test" field
+depends: [
+  "ocaml" {>= "4.05.0"}
+  "dune"  {>= "1.4" }
+  "ocamlfind" { build }
```

Comment:
Is this actually used? From a brief search it did not seem the case

Reply:
The installation seems fine. No library is installed, just executables, so there is no need to address the last two points if this is expected

Reply:
If you mean is `ocamlfind` used ? The answer is that `ocamlfind` is used to build a component that is not compiled by default. As a consequence, teh dependency on `ocamlfind` can probably be removed. I'll try